cmd/clusterctl/cmd/delete_cluster.go

```diff
@@ -19,13 +19,11 @@ package cmd
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: I don't think this is necessary.

Reply:
I _think_ this is just goimports being funny :)

Reply:
Yeah confirmed it's `goimports`, also updated to the latest version to make sure.